.github/dependabot.yml

```diff
@@ -10,11 +10,23 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "gradle"
+    directory: "/dependencyManagement"
```

Comment:
`dependencyManagement` isn't a standalone project, it's a part of the main project -- shouldn't it already have been configured?

Reply:
the global configuration (for `"/"`) only applies to `dependency-name: "com.gradle*"`

we can't apply globally for all dependencies because we use too many old versions intentionally for testing the instrumentation modules

so I think this is a good compromise (and note that dependabot is not very gradle smart, it's just regex parsing `build.gradle` and `build.gradle.kts` files in those directories, I did some spelunking in the dependabot issue)